### PR TITLE
Dw 3794 secrets manager

### PR DESCRIPTION
**Pull request by ChrisScottThomas**
Concourse can now use Secrets Manager.

### CODE REVIEW

File: aviator.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  to: aviator_pipeline.yml
+  - base: ci/meta.yml
+    prune:
+      - meta
+    merge:
+      - with_in: ci/
+        regexp: ".*yml"
+      - with_in: ci/jobs/
+        except:
+          - management-dev.yml.j2
```
Comment (dan-hill2802) [inline aviator.yml:10]: Can the regex not be altered to ignore Jinja files?

Comment (dan-hill2802) [reply aviator.yml:10]: Actually, you can just change the files back to `.yml`, the only things Jinja is putting in is the account numbers, which are already available as variables
https://github.com/dwp/dataworks-admin-utils/blob/master/ci/ami-cleanup/meta.yml#L19

File: terraform/modules/concourse_web/iam.tf [hunk 2]
```diff
@@ -32,20 +32,55 @@ data "aws_iam_policy_document" "web" {
...
+    effect = "Allow"
+
+    actions = [
+      "secretsmanager:ListSecrets",
+      "secretsmanager:GetSecretValue",
+      "secretsmanager:DescribeSecret"
+    ]
+
+    resources = [
+      "arn:aws:secretsmanager:::secret:/concourse/*",
```
Comment (dan-hill2802) [inline terraform/modules/concourse_web/iam.tf:46]: are the secrets manually created and populated?

Comment (mattburgess) [reply terraform/modules/concourse_web/iam.tf:46]: They are at the moment. There's another ticket in the backlog to put a workflow around secrets held in secret manager. At that point this ARN can be changed to reference that TF resource.

Comment (ChrisScottThomas) [reply terraform/modules/concourse_web/iam.tf:46]: We've also left SSM access in.  We need to test if we can lift and shift pipelines and migrate to SM after, or if we need to migrate to SM right away.

File: aviator.yml (hunk 1 above)
Comment (ChrisScottThomas) [reply aviator.yml]: I assume they're all in SSM, which isn't configured as a Credentials Manger in AWS-Concourse.  We'd need to transfer the desired content over to SM, to do that, which isn't in the scope of this work.

**Pull request merged**